summer-of-code/week-11/day-2.md

```diff
@@ -27,6 +39,15 @@ Triggers dont involve physics collisions. They just recognize when two objects a
 
 When a trigger collision is recognized we can call any function that we want through scripting.
 
+### Changing an Object parent
+
+Every object has a parent we can access to. We see the parent of an object in the hierarchy as it apperears that some objects are inside another objects; we'll say from now: Object A is the parent/ a child of object B. We can access the parent property inside any Monobehaviour funcion:
```

Comment:
another objects; > other objects.
we'll say from now: > We say 
parent/ a child > parent/child
funcion > function